content/influxdb/v2.0/reference/flux/stdlib/contrib/alerta/alert.md

```diff
@@ -0,0 +1,173 @@
+---
+title: alerta.alert() function
+description: >
+  The `alerta.alert()` function sends an alert to Alerta.
+menu:
+  influxdb_2_0_ref:
+    name: alerta.alert
+    parent: Alerta
+weight: 202
+---
+
+The `alerta.alert()` function sends an alert to [Alerta](https://www.alerta.io/).
+
+```js
+import "contrib/bonitoo-io/alerta"
+
+alerta.alert(
+  url: "https://alerta.io:8080/alert",
+  apiKey: "0Xx00xxXx00Xxx0x0X",
+  resource: "example-resource",
+  event: "Example event",
+  environment: "",
+  severity: "critical",
+  service: [],
+  group: "",
+  value: "",
+  text: "",
+  tags: [],
+  attributes: {},
+  origin: "InfluxDB",
+  type: "",
+  timestamp: now(),
+)
+```
+
+## Parameters
+
+### url
+({{< req >}})
+Alerta URL.
+
+_**Data type:** String_
+
+### apiKey
+({{< req >}})
+Alerta API key.
+
+_**Data type:** String_
+
+### resource
+({{< req >}})
+Resource associated with the alert.
+
+_**Data type:** String_
+
+### event
+({{< req >}})
+Event name.
+
+_**Data type:** String_
+
+### environment
+Alert environment.
+Default is `""`.
+
+_**Data type:** String_
+
+**Valid values:**
+
+- `""`
+- `"Production"`
+- `"Development"`
+
+### severity
+({{< req >}})
+Event severity.
+See [Alerta severities](https://docs.alerta.io/en/latest/api/alert.html#alert-severities).
+
+_**Data type:** String_
+
+### service
+List of affected services.
```

Comment:
Alerta services?

Reply:
This would be a list of services the function is alerting on and sending to Alerta. These aren't Alerta services.